### PR TITLE
Update logos

### PR DESCRIPTION
**Pull request by julianocosta89**
This PR updates the Cloud Native Linz logo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
+# Cloud Native Linz Website
 
 ![GitHub last commit](https://img.shields.io/github/last-commit/cloudnativelinz/cloudnativelinz.github.io?label=updated)
 ![Website](https://img.shields.io/website?url=https%3A%2F%2Fcloudnativelinz.github.io)![GitHub contributors](https://img.shields.io/github/contributors/cloudnativelinz/cloudnativelinz.github.io)
 [![Twitter Follow](https://img.shields.io/twitter/follow/cloudnativelinz?style=social)](https://twitter.com/cloudnativelinz)
 
 ## Visit the website
 
-Find the website here: https://cloudnativelinz.github.io.
+Find the website here: <https://cloudnativelinz.github.io>.
```
Comment (jetzlstorfer) [inline README.md:9]: I am not familiar with the <http..> syntax and used it without the brackets until now. Could you please let me know on the reasons and advantages of this syntax?

**Review by jetzlstorfer — COMMENTED** [review]
clarifying minors in the readme

File: README.md (hunk 1 above)
Comment (jetzlstorfer) [reply README.md:9]: Interestingly, I just learned there are multiple ways to display URLs in markdowns: https://meta.stackexchange.com/questions/13501/links-to-urls-containing-parentheses

Comment (julianocosta89) [reply README.md:9]: I have a MD plugin that complains about bare-urls.
Here is the explanation of it:
https://github.com/DavidAnson/markdownlint/blob/v0.31.1/doc/md034.md

Comment (jetzlstorfer) [reply README.md:9]: ok, fine for me :) thank you for clarification

**Review by jetzlstorfer — APPROVED** [review]

**Pull request merged**